### PR TITLE
Chef Automate rename

### PR DESCRIPTION
**Pull request by kamarja**
- Removed reference to old submodule

### CODE REVIEW

**Comment by santiagocardenas** [comment]
Missing updates to config.yml and move of parameter files into ci/ directory.

**Comment by kamarja** [comment]
Oops. Ok, done.

**Comment by santiagocardenas** [comment]
Two more things:

- Readme also needs naming updates
- (although separate) can we use the VPC Quick Start in this?

File: ci/chef-automate-quickstart.json [hunk 1]
```diff
@@ -9,7 +9,7 @@
     },
     {
         "ParameterKey": "QSS3KeyPrefix",
-        "ParameterValue": "chef/chef-server/latest/"
+        "ParameterValue": "chef/automate/latest/"
```
Comment (santiagocardenas) [inline ci/chef-automate-quickstart.json]: Please remove QSS3BucketName and QSS3KeyPrefix from the .json parameter file.

File: .gitmodules [hunk 2]
```diff
@@ -1,3 +1,6 @@
-[submodule "microsoft-utilities"]
-	path = microsoft-utilities
+[submodule "submodules/quickstart-microsoft-utilities"]
```
Comment (santiagocardenas) [inline .gitmodules]: Each submodule needs to explicitly track master. See other Quick Starts.

File: ci/chef-automate-quickstart.json [hunk 3]
```diff
@@ -3,14 +3,6 @@
...
-        "ParameterKey": "QSS3BucketName",
-        "ParameterValue": "quickstart-reference"
-    },
-    {
-        "ParameterKey": "QSS3KeyPrefix",
-        "ParameterValue": "chef/chef-server/latest/"
-    },
     {
         "ParameterKey": "RemoteAdminCIDR",
         "ParameterValue": "0.0.0.0/0"
```
Comment (santiagocardenas) [inline ci/chef-automate-quickstart.json]: Make this the same value as VPCCIDR

File: ci/chef-automate-quickstart.json [hunk 4]
```diff
@@ -3,14 +3,6 @@
         "ParameterKey": "KeyPairName",
         "ParameterValue": "cikey"
```
Comment (santiagocardenas) [inline ci/chef-automate-quickstart.json]: make this $[alfred_getkeypair]

Comment (santiagocardenas) [reply ci/chef-automate-quickstart.json]: Also, add all other parameters, except bucke tname and key prefix. Include generators where possible.

File: templates/chef-automate.template [hunk 5]
```diff
@@ -1,68 +1,67 @@
...
+                },
                 "ChefAutomateInstanceType": {
                     "default": "Chef Automate Instance Type"
                 },
-                "SubnetId": {
-                    "default": "Public Subnet ID"
+                "IncludeDemoInstances": {
+                    "default": "Enable and Launch Demo Instances."
+                },
+                "PublicSubnetCIDR": {
```
Comment (santiagocardenas) [inline templates/chef-automate.template]: This should just be SubnetCIDR

File: templates/chef-automate.template [hunk 6]
```diff
@@ -151,6 +113,43 @@
...
+            "AllowedPattern": "^[0-9a-zA-Z-/]*$",
+            "ConstraintDescription": "Quick Start key prefix can include numbers, lowercase letters, uppercase letters, hyphens (-), and forward slash (/).",
+            "Default": "chef/automate/latest/",
+            "Description": "S3 key prefix for the Quick Start assets. Quick Start key prefix can include numbers, lowercase letters, uppercase letters, hyphens (-), and forward slash (/).",
+            "Type": "String"
+        },
+        "RemoteAdminCIDR": {
+            "Description": "CIDR Block or IP for SSH and RDP access",
+            "Type": "String",
+            "AllowedPattern": "[a-zA-Z0-9]+\\..+"
```
Comment (santiagocardenas) [inline templates/chef-automate.template]: This is a bad pattern. You want an IP address pattern like we do in other Quick Starts. Same goes to other CIDR blocks (but be careful if copying from other Quick Starts as those limit subnet or VPC sizes to supported ones)

**Pull request merged**